public/js/graph/react_node.js.jsx

```diff
@@ -151,28 +91,52 @@ var ReactSVG = React.createClass({
 var ReactRegionLabels = React.createClass({
     getInitialState: function () {
         return {
-            labelsList: []
+            labelsJSON: []
         };
     },
 
     componentDidMount: function () {
```

Comment:
Do this in `componentWillMount`. Don't check if the component is mounted... you should be able to set the state before mounting.